packages/transaction-log/src/services/api.service.ts

```
@@ -80,3 +81,47 @@ export async function getServerHeartBeat(): Promise<{
+    }
+    await addToTaskQueue(payload)
+  } catch (error) {
+    throw error
+  }
+}
+
+export async function addTransactionsToReRunQueueByFilters(params: {
+  batchSize: number
+  filterLimit: number
+  filterPage: number
+  filters: {}
```

Biome

[error] Don't use '{}' as a type.

Prefer explicitly define the object shape. '{}' means "any non-nullable value".

(lint/complexity/noBannedTypes)